src/test/java/ContactManagerTest.java

```diff
@@ -0,0 +1,31 @@
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class ContactManagerTest {
+
+    ContactManager contactManager;
+    Contact contact;
+    @Before
+    public void initialize() {
```

Comment:
This is very ruby. I think `setUp` is the the java convention. Check on junit docs. I might wrong.